core/src/tile/tileWorker.cpp

```diff
@@ -144,6 +185,10 @@ void TileWorker::setScene(std::shared_ptr<Scene>& _scene) {
     }
 }
 
+void TileWorker::notifyAll() {
+    m_condition.notify_all();
+}
+
```

Comment:
Since tasks are requeued within the worker if a dependent task is not ready, we need to inform/notify the worker to run again on the "conditions" if one of the dependent tasks finishes.

so notifyAll is called from rasterCallback!

Also, its possible workers are done and waiting for a notification, when a task was cancelled, so all workers are notified on a task cancel from the main thread. (This might have some redundant notifications when a task was cancelled from the main thread and workers are not waiting on anything but are in middle of building a tile)